lib/yaml_elixir/mapper.ex

```diff
@@ -19,6 +19,13 @@ defmodule YamlElixir.Mapper do
   defp _to_map({:yamerl_map, :yamerl_node_map, _tag, _loc, map_tuples}, options),
     do: _tuples_to_map(map_tuples, empty_container(options), options)
 
+  defp _to_map({:yaml_elixir_keyword_list, _module, _tag, _loc, tuples}, options) do
+    case _tuples_to_map(tuples, empty_container(options), options) do
+      map = %{} -> for {key, value} <- map, do: {key, value}
```

Comment:
I suggest to avoid `case` here:

```elixir
defp _to_map({:yaml_elixir_keyword_list, _module, _tag, _loc, tuples}, options) do
  tuples
  |> _tuples_to_map(empty_container(options), options)
  |> to_keyword_list()
end
```

and then define a proper `to_keyword_list/1` with guards like `when is_map(map)` for example.

Reply:
Done